packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/activationMode.test.ts

```diff
@@ -4,24 +4,26 @@ import * as vscode from 'vscode';
 import sinon, { stubInterface, stubObject } from 'ts-sinon';
 import * as sinonChai from 'sinon-chai';
 import { shared as lspCommon } from '@salesforce/lightning-lsp-common';
-import { MockContext } from './MockContext';
+import { MockExtensionContext } from './MockExtensionContext';
```

Comment:
Just checking that the directory was renamed here as well? This was the case for all of the MockContext's?

Reply:
No, just the file was renamed.  The path is `packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts`, and the parent directory, `activation` was not changed, just the filename.